src/main/java/GeneralService/OnlineServiceHandler.java

```diff
@@ -0,0 +1,11 @@
+package GeneralService;
+
+import java.util.List;
+
+public class OnlineServiceHandler {
+    private List<String> services;
+
+    public OnlineServiceHandler(List<String> services) {
```

Comment:
I want this List to be of type Service.
This list should also be autowired by spring.